app/services/notification_service.rb

```diff
@@ -0,0 +1,32 @@
+class NotificationService
+  attr_accessor :users
+
+  def initialize(matching_users)
+    @users = matching_users
+  end
+
+  def notify
+    ids = @users.map(&:player_id)
+    params = request_params(ids)
+    uri = URI.parse(ENV['ONE_SIGNAL_NOTIFY_URL'].to_s)
+    send_request(uri, params)
+  end
+
+  def request_params(ids)
+    { 'app_id' => ENV['ONE_SIGNAL_APP_ID'],
```

Comment:
use the new syntax for hashes.
Also you can align like:
```
{
  name: 'tiziana',
  age: 24
}
```
Also I think the key does not need the `' '`